README.md

```diff
@@ -131,16 +139,6 @@ Example command:
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/219120861-7d962f32-e44d-4338-bded-361d374a1d57.png)

**[Optional]**

- [ ] You have no commands in the section and sub-sections :fearful: Kindly add the following commands in their respective sections to make your readme file descriptive and professional :+1:

### Setup

```
cd my-folder
git clone git@github.com:Dmambo/coincentral.git
cd coincentral
```

### Install

```
npm install
```

### Usage

```
npm start
```

### Testing

```
npm test
```